Alpha_shapes_3/demo/Alpha_shapes_3/Alpha_shape_3.cpp

```diff
@@ -8,16 +8,28 @@
 
 int main(int argc, char** argv)
 {
- QApplication application(argc,argv);
+  QSurfaceFormat fmt;
```

Comment:
Same question: Why here you don't use code_to_call_before_creation_of_QCoreApplication ?